Assets/HoloToolkit/Utilities/Tests/Scripts/GestureResponder.cs

```diff
@@ -1,11 +1,31 @@
-﻿using UnityEngine;
-using System.Collections;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Nit: Do we know for sure this is copyright by Microsoft? You're not an employee so can't assert that.